2022-01-14 09:49:46,003 INFO    MainThread:125532 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 09:49:46,004 INFO    MainThread:125532 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:49:46,004 INFO    MainThread:125532 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:49:46,004 INFO    MainThread:125532 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_094946-2l1e6stf/logs/debug.log
2022-01-14 09:49:46,004 INFO    MainThread:125532 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_094946-2l1e6stf/logs/debug-internal.log
2022-01-14 09:49:46,005 INFO    MainThread:125532 [wandb_init.py:init():404] calling init triggers
2022-01-14 09:49:46,005 INFO    MainThread:125532 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 09:49:46,005 INFO    MainThread:125532 [wandb_init.py:init():460] starting backend
2022-01-14 09:49:46,005 INFO    MainThread:125532 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 09:49:46,012 INFO    MainThread:125532 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 09:49:46,015 INFO    MainThread:125532 [backend.py:ensure_launched():221] started backend process with pid: 125588
2022-01-14 09:49:46,016 INFO    MainThread:125532 [wandb_init.py:init():469] backend started and connected
2022-01-14 09:49:46,018 INFO    MainThread:125532 [wandb_init.py:init():533] updated telemetry
2022-01-14 09:49:46,027 INFO    MainThread:125532 [wandb_init.py:init():563] communicating current version
2022-01-14 09:49:46,399 INFO    MainThread:125532 [wandb_init.py:init():568] got version response 
2022-01-14 09:49:46,399 INFO    MainThread:125532 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 09:49:46,590 INFO    MainThread:125532 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 09:49:46,646 INFO    MainThread:125532 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 09:49:46,647 INFO    MainThread:125532 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 09:49:46,647 INFO    MainThread:125532 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 09:49:46,648 INFO    MainThread:125532 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 09:49:46,648 INFO    MainThread:125532 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 09:49:46,770 INFO    MainThread:125532 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 09:49:46,771 INFO    MainThread:125532 [wandb_run.py:_restore():1752] restore
2022-01-14 09:49:49,205 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:49:49,431 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:49:49,593 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 2460
}

2022-01-14 09:49:49,694 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4862
}

2022-01-14 09:49:49,795 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:49,897 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:50,000 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:50,104 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:50,208 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:50,710 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:49:51,094 INFO    MainThread:125532 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}
local_info {
}

2022-01-14 09:49:52,182 INFO    MainThread:125532 [wandb_run.py:_append_files():2180] logging synced files
